src/pymor/tools/floatcmp.py

```diff
@@ -31,7 +31,7 @@ def float_cmp(x, y, rtol=1e-14, atol=1e-14):
     atol
         The absolute tolerance.
     """
-    return np.abs(x - y) <= atol + np.abs(y) * rtol
+    return np.abs(x - y) <= atol + np.abs(y * rtol)
```

Comment:
Why this change?

Reply:
This avoids the `RuntimeWarning` when `y=np.inf` and `rtol=0`, because `np.abs` builds a NumPy array. But, I see that another solution might be to replace `np.abs` by `abs`.
Although, neither removes `RuntimeWarning` if `y` is already a NumPy array with infinities. A fix for that might be to ignore `RuntimeWarnings`.

Reply:
Ok. I was wondering if anything happens with negative `rtol`. But I guess there is no good reason for choosing negative tolerances .. - `np.abs` should be better then `abs` as the latter is not vectorized.

Reply:
I guess `abs` refers to `np.abs` for NumPy arrays via `__abs__`.